cdap-ui/app/cdap/components/TriggeredPipelines/index.js

```diff
@@ -154,7 +128,31 @@ export default class TriggeredPipelines extends Component {
   }
 }
 
+TriggeredPipelinesView.propTypes = {
+  pipelineName: PropTypes.string.isRequired,
+  namespace: PropTypes.string.isRequired,
```

Comment:
Are we actually using this anywhere as a prop? I only see one mention of `namespace` variable being used in this component, and we are getting that from the NamespaceStore.